releases/upcoming.rst

```diff
@@ -18,6 +18,8 @@ Fixed
 
 * Fix showing UNKNOWN on ``postgrest --help`` invocation.
   |br| -- `@monacoremo <https://github.com/monacoremo>`_
+* Fix :ref:`_s_procs_variadic`.
```

Comment:
I don't have Sphinx installed, so I can't test that - is that the correct format for the link?

Reply:
Needs the first underscore removed. Sphinx has its quirks.
```suggestion
* Fix :ref:`s_procs_variadic`.
```